### ISSUE TITLE
Evaluate function for online learning

### ISSUE THREAD

**Issue by angelomenezes** [angelomenezes/Data_Science_for_Finance#1]
Function needs to be properly implemented

**Comment by angelomenezes** [comment]
maybe, after saving the model, this might be a temporary solution:

from keras.models import model_from_json

def sav_model(model):    
    json_string = model.to_json()
    open('model.json', 'w').write(json_string)
    model.save_weights('weights.h5', overwrite=True)
    
def lad_model():
    model = model_from_json(open('model.json').read())
    model.load_weights('weights.h5')
    #sgd = SGD(lr = 0.1, momentum = 0.9, decay = 0, nesterov = False)
    adam = Adam(lr=0.01, beta_1=0.9, beta_2=0.999, epsilon=None, decay=0.0, amsgrad=False)
    model.compile(optimizer=adam, loss = 'mse')
    return model
